### ISSUE TITLE
UIScreenEdgePanGestureRecognizer Instead of the UIPanGestureRecognizer

### ISSUE THREAD

**Issue by mohamede1945** [showmecode/CHXSwipeBackPopAnimator#1]
I think it's better to use `UIScreenEdgePanGestureRecognizer` instead of the `UIPanGestureRecognizer`.

**Comment by cuzv** [comment]
Yep. 
When I wrote this stuff I didn't realized there is a gesture recognizer called `UIScreenEdgePanGestureRecognizer`.
It will be replaced with `UIScreenEdgePanGestureRecognizer` late. :]

**Comment by mohamede1945** [comment]
Thank you! Also, it would be great if you could make the animator and other properties public so that one can customize them.

**Comment by cuzv** [comment]
Hey buddy, wait a minute. It is trying to make swipe back pan gesture works on all region, not just screen edge. I wasn't thinking when you told me `UIScreenEdgePanGestureRecognizer` thing, just thought it was really good.
So, sorry I can't do that. :joy:

**Comment by mohamede1945** [comment]
I understand. But currently, this library doesn't work if you have a `UITableView` or any `UIScrollView`.

**Comment by cuzv** [comment]
That's not true. I wrote this library for a project, it worked. can you provide a little demo?
